packages/api-database/source/migrations/1697617471901-CreateIndexes.ts

```diff
@@ -0,0 +1,93 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class CreateIndexes1697617471901 implements MigrationInterface {
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		// language=postgresql
+		await queryRunner.query(`
+            CREATE UNIQUE INDEX transactions_sender_nonce ON transactions(sender_public_key, nonce);
+            CREATE INDEX transactions_recipient_id ON transactions(recipient_id);
+            CREATE INDEX transactions_sender ON transactions(sender_public_key);
+
+            CREATE INDEX transactions_block_id ON transactions(block_id);
+            CREATE INDEX transactions_block_height ON transactions(block_height, sequence);
```

Comment:
Sequence is not in index name. 

```suggestion
            CREATE INDEX transactions_block_height ON transactions(block_height);
```